index.html

```diff
@@ -69,6 +69,39 @@
       </div>
     </div>
 
+    <!-- APD community service sites filters -->
+    <div class="row">
+      <div class="col-xs-12">
+        <p><strong>Filter for APD community service sites.</strong> You cannot filter by age, language, fcaility, programs, or payment preferences.</p>
```

Comment:
fcaility --> facility

I think there should be an explanation of what an APD community service site is—at least, I don't know what it is, and didn't find any illuminating answers on google. do you feel it is something that is obvious though?

Reply:
APD stands for ["Adult Probation Department"](http://www.cookcountycourt.org/ABOUTTHECOURT/OfficeoftheChiefJudge/ProbationDepartments/ProbationforAdults/AdultProbationDepartment.aspx).  

And now...an important note! This site can be accessed by anyone, but its principally used as an internal tool in the Cook County probation department. 

To your comment, I don't think we need APD as a descriptor, at all. (I simply included it, because that's how our client labeled the resources in the spreadsheet. It's not very meaningful, otherwise.)